sass/base/components/just-in-time-block/_just-in-time.scss

```diff
@@ -4,33 +4,43 @@
 //
 // Style guide: 20.1004
 
+$colour-scorpion: #595959;
+
 .form__row--just-in-time-block {
 
   .toggle-link {
     position: relative;
     font-size: 18px;
-    border-bottom: 1px solid #666;
-    font-family: $font-medium;
-    color: #666;
+    border-bottom: 4px solid $color-scorpion;
+    font-family: $font-bold;
+    color: $color-scorpion;

```

Comment:
you need to update the colour variables here too

Reply:
ah thanks